main.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
```

Comment:
[Optional] Kindly link the html and css by adding a `link` tag to this html file like so 
`<head><link rel="stylesheet" href="styles.css"></head>`
You can read more on this [here](https://www.w3schools.com/tags/tag_link.asp)